tests/testfiles/lunamark-markdown/fenced-divs.test

```diff
@@ -184,7 +184,7 @@ BEGIN fencedDivAttributeContext
 attributeIdentifier: some-identifier
 interblockSeparator
 blockQuoteBegin
-interblockSeparator
+paragraphSeparator
```

Comment:
Why is this expected?

Here is the input:

``` md
::: {#some-identifier}
This is the beginning of a div

> This is a blockquote that contains three colons:
> :::

This is the end of a div
:::
```

I don't see another paragraph inside the blockquote.

Here is what I get by running `markdown-cli` at commit 7e348f7 from this PR:

``` sh
$ docker run --rm -it ghcr.io/witiko/markdown:7e348f73-latest-no_docs
$ cat > above-markdown-text.md
::: {#some-identifier}
This is the beginning of a div

> This is a blockquote that contains three colons:
> :::

This is the end of a div
:::
^D
$ markdown-cli fencedDivs=true < above-markdown-text.md
\markdownRendererDocumentBegin
\markdownRendererFencedDivAttributeContextBegin
\markdownRendererAttributeIdentifier{some-identifier}This is the beginning of a div\markdownRendererInterblockSeparator
{}\markdownRendererBlockQuoteBegin
This is a blockquote that contains three colons:\markdownRendererParagraphSeparator
{}:::
\markdownRendererBlockQuoteEnd \markdownRendererInterblockSeparator
{}This is the end of a div\markdownRendererFencedDivAttributeContextEnd \markdownRendererDocumentEnd
```

Here, I would expect a `\markdownRendererSoftLineBreak` instead of `\markdownRendererParagraphSeparator` between "colons:" and ":::".

Reply:
In commit dda4a43d from this PR, I updated the test file to reflect the actual expected output.

Reply:
On a reflection, this seems to have been broken ever since #226. In commit aa1cedef from this PR, I reverted my previous commit and let's leave this until a future ticket.